cli/command/formatter/trust.go

```diff
@@ -133,3 +133,17 @@ func (c *signerInfoContext) Keys() string {
 func (c *signerInfoContext) Signer() string {
 	return c.s.Name
 }
+
+type SignerInfoList []SignerInfo
```

Comment:
sorry, I forgot to include a comment in my snippet -- lint might complain about this